### PR TITLE
Made a few small changes

### PR DESCRIPTION
**Pull request by knobunc**
- Got the 'oc routes' output
 - Supressed the host check in ssh
 - Unified common code between host and node
 - Cleaned up the formatting to make the commands being run a little more obvious

### CODE REVIEW

File: hack/debug.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# We want to supress the hostkey check
+SSH_OPTS='-o StrictHostKeyChecking=no'
```
Comment (danwinship) [inline hack/debug.sh]: might as well put the `-o PasswordAuthentication=no` in here too. And move it to run_self_via_ssh(); nothing else uses it

File: hack/debug.sh [hunk 2]
```diff
@@ -34,6 +37,29 @@ filter_env () {
     awk '/"env": \[$/ { indent = index($0, "\""); skipping = 1; next } !skipping { print; } skipping { if (index($0, "]") == indent) skipping = 0; }'
 }
 
+log_system () {
+    logpath=$1
+
+    echo_and_eval  journalctl --boot                                  &> $logpath/journal-full
+    echo_and_eval  journalctl -u $aos_master_service                  &> $logpath/journal-openshift
+    echo_and_eval  systemctl show $aos_master_service                 &> $logpath/systemctl-show
```
Comment (danwinship) [inline hack/debug.sh]: Note that this needs to be `$aos_master_service` for the master, but `$aod_node_service` for the node

File: hack/debug.sh [hunk 3]
```diff
@@ -45,20 +71,15 @@ do_master () {
...
-    echo_and_eval oc get nodes -o json >& $logmaster/nodes
-    echo_and_eval oc get pods --all-namespaces -o json | filter_env >& $logmaster/pods
-    echo_and_eval oc get services --all-namespaces -o json >& $logmaster/services
+
+    # Log the generic system stuff
+    log_system $logmaster
+
+    # And the master specific stuff
+    echo_and_eval  oc get nodes                     -o json           &> $logmaster/nodes
+    echo_and_eval  oc get pods     --all-namespaces -o json           &> filter_env >& $logmaster/pods
```
Comment (danwinship) [inline hack/debug.sh]: `| filter_env` not `&> filter_env`

File: hack/debug.sh [hunk 4]
```diff
@@ -179,7 +200,7 @@ do_pod_to_pod_connectivity_check () {
     echo_and_eval ovs-appctl ofproto/trace br0 "in_port=${other_pod_port},reg0=${other_pod_vnid},ip,nw_src=${other_pod_addr},nw_dst=${base_pod_addr},dl_dst=${base_pod_ether}"
     echo ""
 
-    if nsenter -n -t $base_pod_pid ping -c 1 -W 2 $other_pod_addr >& /dev/null; then
+    if nsenter -n -t $base_pod_pid  ping -c 1 -W 2 $other_pod_addr  &> /dev/null; then
```
Comment (danwinship) [inline hack/debug.sh]: unnecessary added whitespace?

File: hack/debug.sh [hunk 5]
```diff
@@ -209,7 +230,7 @@ do_pod_external_connectivity_check () {
     echo_and_eval ovs-appctl ofproto/trace br0 "in_port=2,ip,nw_src=198.51.100.1,nw_dst=${base_pod_addr},dl_dst=${base_pod_ether}"
     echo ""
 
-    if nsenter -n -t $base_pod_pid ping -c 1 -W 2 www.redhat.com >& /dev/null; then
+    if nsenter -n -t $base_pod_pid  ping -c 1 -W 2 www.redhat.com  &> /dev/null; then
```
Comment (danwinship) [inline hack/debug.sh]: likewise

File: hack/debug.sh (hunk 4 above)
Comment (danwinship) [inline hack/debug.sh]: ah, "to make the command being run more obvious"; we should probably use `--` between the nsenter args and the command anyway

**Comment by dcbw** [comment]
I have no comments above and beyond danw's.

File: hack/debug.sh [hunk 6]
```diff
@@ -45,20 +71,15 @@ do_master () {
...
-    echo_and_eval oc get services --all-namespaces -o json >& $logmaster/services
+
+    # Log the generic system stuff
+    log_system $logmaster
+
+    # And the master specific stuff
+    echo_and_eval  oc get nodes                     -o json           &> $logmaster/nodes
+    echo_and_eval  oc get pods     --all-namespaces -o json           &> filter_env >& $logmaster/pods
+    echo_and_eval  oc get services --all-namespaces -o json           &> $logmaster/services
+    echo_and_eval  oc get routes   --all-namespaces -o json           &> $logmaster/aos_routes
```
Comment (pravisankar) [inline hack/debug.sh]: Can we also capture output of these commands:
oc get clusternetwork -o json
oc get hostsubnets -o json
oc get netnamespaces -o json

**Comment by dcbw** [comment]
LGTM

**Comment by pravisankar** [comment]
lgtm

**Pull request merged**